app/uk/gov/hmrc/channelpreferences/model/AgentEnrolment.scala

```diff
@@ -18,7 +18,7 @@ package uk.gov.hmrc.channelpreferences.model
 
 import play.api.libs.json.{ Json, OFormat }
 
-final case class AgentEnrolment(arn: String, itsaId: String, nino: Option[String], sautr: Option[String])
```

Comment:
is it possible that these two wouldn't be present at all from your retrievals?

Reply:
Is channel-preferences using new collection. It may not be happy with existing data?

Reply:
@elabeca I didnt completely understand your question, but I can say that they should be present as part of the request to be sent for the stub, so hope we dont see the case where they are present as part of this enrolment..

Reply:
@elabeca @muralishub  It is not being used for the retrievals, it is just for validating the request. We dont have any data store in that format.

Reply:
@satyajosyula2020 if they are meant to be present for the request, does that mean we return a 400? if we do, is there a test for that?

Reply:
@elabeca yes, it should return 400. Added tests now. Please review it again. thanks..

Reply:
ah ok thanks @satyajosyula2020